### PR TITLE
Adds tests for kad replacement and restores kad operator configs

### PR DESCRIPTION
**Pull request by jenlij**
What/Why:
• Loops through all satellites and storage nodes in the testplanet setup test to make sure all nodes and satellites can reach each other
• Tests that an untrusted satellite cannot request info from a node
• Restores the original kademlia config operator wallet and email to allow for a better UX for backwards compatibility

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: internal/testplanet/storagenode.go [hunk 1]
```diff
@@ -71,9 +72,11 @@ func (planet *Planet) newStorageNodes(count int, whitelistedSatellites storj.Nod
...
 				},
 			},
-			Operator: storagenode.OperatorConfig{
-				Email:  prefix + "@mail.test",
-				Wallet: "0x" + strings.Repeat("00", 20),
+			Kademlia: kademlia.Config{
+				Operator: kademlia.OperatorConfig{
+					Email:  prefix + "@mail.test",
+					Wallet: "0x" + strings.Repeat("00", 20),
+				},
```
Comment (JessicaGreben) [inline internal/testplanet/storagenode.go]: are we only adding this back to the storagenode? but not the satellite? for ex, originally the kad.operator.config was in both https://github.com/storj/storj/pull/2966/files#diff-7a0c41e7d3651613555d7d24a6d785dbL89

Comment (cam-a) [reply internal/testplanet/storagenode.go]: since we run the satellites, I think this should be okay

Comment (cam-a) [reply internal/testplanet/storagenode.go]: As far as I know, it's okay to completely remove kademlia from the satellite configs. I think the issue was that by moving the operator configs, the SNOs would have to reconfigure

File: internal/testplanet/planet_test.go [hunk 2]
```diff
@@ -38,18 +38,22 @@ func TestBasic(t *testing.T) {
...
+				node := sn.Local()
+				conn, err := sn.Transport.DialNode(ctx, &satellite)
+				require.NoError(t, err)
+				_, err = pb.NewNodeClient(conn).CheckIn(ctx, &pb.CheckInRequest{
+					Address:  node.GetAddress().GetAddress(),
+					Version:  &node.Version,
+					Capacity: &node.Capacity,
+					Operator: &node.Operator,
+				})
+				require.NoError(t, err)
```
Comment (JessicaGreben) [inline internal/testplanet/planet_test.go]: beautiful!

**Pull request merged**